setup.py

```diff
@@ -21,7 +21,8 @@
               'bluesky.callbacks'],
     package_data={'bluesky': ['schema/*.json']},
     install_requires=['jsonschema', 'traitlets', 'prettytable', 'cycler',
-                      'numpy', 'matplotlib', 'super_state_machine'],
+                      'numpy', 'matplotlib', 'super_state_machine',
+                      'historydict'],
```

Comment:
@tacaswell I don't think that's right. I think you are thinking of `setup_requires`. The terminology is awful, but I believe `install_requires` the most closely means "runtime requirements." [Ref](https://pythonhosted.org/setuptools/setuptools.html#new-and-changed-setup-keywords)